### PR TITLE
Feat/add new components

### PR DESCRIPTION
**Pull request by diego-morales-flores-1996**
Adding new components here is the list:

- Notification
- StepProgressBar
- Status
- TooltipMap

### CODE REVIEW

File: src/components/elements/Status/Status.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    const colorMap: { [key: string]: string } = {
+      Approved: "text-green-500",
+      Submitted: "text-blue",
+      Draft: "text-pinkCustom",
+      "Under Review": "text-tertiary-600",
+      "Needs More Info": "text-tertiary-600",
+      "Planting in Progress": "text-blue"
+    };
+
+    return colorMap[status] || "";
```
Comment (roguenet) [inline src/components/elements/Status/Status.tsx:28]: `??`

File: src/components/elements/Status/Status.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      Submitted: "bg-blue-200",
+      Draft: "bg-pinkCustom-200",
+      "Under Review": "bg-tertiary-50",
+      "Awaiting approval": "bg-tertiary-50",
+      "Needs More Info": "bg-tertiary-50",
+      "Needs more information": "bg-tertiary-50",
+      "Planting in Progress": "bg-blue-200"
+    };
+
+    return colorMap[status] || "";
```
Comment (roguenet) [inline src/components/elements/Status/Status.tsx:43]: `??`

File: src/components/elements/Status/Status.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      className={tw("flex items-center justify-center rounded-xl py-1 px-[6px]", getColorStatusBg(status), className)}
+    >
+      <Text
+        variant={textVariant}
+        className={`flex w-fit items-center gap-[6px] whitespace-nowrap ${getColorStatusText(status)}`}
+      >
+        <When condition={status === "Approved"}>
+          <Icon name={IconNames.CHECK_CIRCLE_FILL} className="h-4 w-4 text-secondary" />
+        </When>
+        {status}
```
Comment (roguenet) [inline src/components/elements/Status/Status.tsx:57]: Wrap in `t` from `useT` please.

File: src/components/elements/TooltipMap/TooltipMap.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+          </Text>
+        </div>
+      </div>
+
+      <hr className="my-2 border border-grey-750" />
+      <div className="flex w-full items-center justify-center">
+        <button className="flex items-center justify-center gap-1" onClick={setEditPolygon}>
+          <Icon name={IconNames.CLICK} className="h-4 w-4" />
+          <Text variant="text-10-light" className="italic text-black">
+            click to see polygon details
```
Comment (roguenet) [inline src/components/elements/TooltipMap/TooltipMap.tsx:84]: All the text in this component needs to be wrapped in `t` from `useT`

File: src/components/elements/Notification/Notification.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        <When condition={type === "warning"}>
+          <Icon name={IconNames.IC_WARNING} width={24} height={24} />
+        </When>
+      </div>
+      <div>
+        <Text variant="text-bold-body-300" className={TextClasses}>
+          {title}
+        </Text>
+        <Text variant="text-body-200" className="mt-2 !font-primary">
+          {message}
```
[the remark this answers is not in the document]
Comment (diego-morales-flores-1996) [reply src/components/elements/Notification/Notification.tsx:61]: done

File: src/components/elements/Status/Status.tsx (hunk 1 above)
Comment (diego-morales-flores-1996) [reply src/components/elements/Status/Status.tsx:28]: done

File: src/components/elements/Status/Status.tsx (hunk 2 above)
Comment (diego-morales-flores-1996) [reply src/components/elements/Status/Status.tsx:43]: done

File: src/components/elements/Status/Status.tsx (hunk 3 above)
Comment (diego-morales-flores-1996) [reply src/components/elements/Status/Status.tsx:57]: done

File: src/components/elements/TooltipMap/TooltipMap.tsx (hunk 4 above)
Comment (diego-morales-flores-1996) [reply src/components/elements/TooltipMap/TooltipMap.tsx:84]: done

**Review by roguenet — APPROVED** [review]

**Pull request merged**